packages/slice-machine/lib/utils/slices.ts

```diff
@@ -0,0 +1,39 @@
+import glob from "glob";
+
+import { getFormattedLibIdentifier } from "@slicemachine/core/build/libraries";
+import { SliceSM } from "@slicemachine/core/build/models/Slice";
+
+import Files from "./files";
+import { CustomPaths } from "../models/paths";
+import * as IO from "../io";
+
+export function getLocalSlices(
+  cwd: string,
+  libraries: string[] = []
+): SliceSM[] {
+  const slices: SliceSM[] = [];
+  for (const library of libraries) {
```

Comment:
Mixing loops and iterative methods, for consistency is it posable to just iterative methods?